src/Core/src/Platform/Android/MauiScrollView.cs

```diff
@@ -60,6 +60,11 @@ public void SetHorizontalScrollBarVisibility(ScrollBarVisibility scrollBarVisibi
 			{
 				scrollBarVisibility = _defaultHorizontalScrollVisibility;
 			}
+			else
+			{
+				// Only disable fading when we explicitly set visibility to Always.
+				_hScrollView.ScrollbarFadingEnabled = scrollBarVisibility != ScrollBarVisibility.Always;
```

Comment:
This looks like it is never reset if you do default. What happens if you move this out of the else and next to the line below:

```cs
_hScrollView.HorizontalScrollBarEnabled = scrollBarVisibility == ScrollBarVisibility.Always
```

I think it should still work.